tests/multiparser_js_css/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -158,4 +188,35 @@ styled.div\`
   }
 \`;
 
+styled.div\`
+  \${bar}
+  baz;
+\`;
+
+styled.span\`
+  foo
+  \${bar}
+  baz
+\`;
+
+styled.div\`
+  foo
+  \${bar}
+  \${baz}
+\`;
+
+styled.span\`
+  \${foo}
+  \${bar};
+\`;
+
+styled.div\`
+  \${foo} bar;
+\`;
+
+styled.span\`
+  \${foo} \${bar}
+  baz: \${foo}
+\`;
```

Comment:
Let's add test with semicolon on end:
```js
styled.span`
${foo};
${bar};
`
```

```js
styled.span`
${foo}: ${bar};
`
```

```js
styled.span`
${foo}: ${bar}
`
```

And with newline(s) between `:` and `;`.

Reply:
Updated :+1: